tests/ui/test_ui_tools.py

```diff
@@ -1302,13 +1302,21 @@ def test_keypress_edit_message(self, mocker, message_fixture,
             <p>C</p>""", "░ A\n░ B\n\nC"),
         ("""<blockquote>
                 <p><a href='https://chat.zulip.org/'</a>czo</p>
-            </blockquote>""", "░ [czo](https://chat.zulip.org/)\n")
+            </blockquote>""", "░ [czo](https://chat.zulip.org/)\n"),
+        pytest.param("""<blockquote>
+                            <blockquote>
+                                <p>A<br>
+                                B</p>
+                            </blockquote>
+                        </blockquote>
+            """, "░ ░ A\n░ ░ B", marks=pytest.mark.xfail(reason="nested bq"))
```

Comment:
I would give the reason as more like "rendering bug", rather than the id?

Reply:
Done :+1: